owa/app/components/admissionLocation/admissionLocationWrapper.js

```diff
@@ -42,12 +42,7 @@ export default class AdmissionLocationWrapper extends React.Component {
 
     fetchAllAdmissionLocations(self) {
         axios
-            .get(this.urlHelper.apiBaseUrl() + '/location', {
-                params: {
-                    tags: 'Admission Location',
-                    v: 'full'
-                }
-            })
+            .get(this.urlHelper.apiBaseUrl() + '/location?tag=Admission%20Location&v=full')
```

Comment:
So, as far as I can tell, this and the call below never worked.  The command above ended up getting implemented as a REST query that looked like `...?tags=Admission+Location`, and that `Admission+Location` was getting treated literally.  The `tags` property was also incorrect it seems, with a correct property of `tag`.  I needed to make this change in order to get the OWA to work properly so I could set up data for testing and confirm my fixes actually work.